index.html

```diff
@@ -5,43 +5,179 @@
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Join</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Sponsor</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">News</a>
+          </li>
+
+          <li class="list-items">
+            <a href="#">WITMP 2023</a>
+          </li>
         </ul>
-      </div>
+      </nav>
+  </header>
+<!-- Header Ends Here -->
+
+<!-- Main Starts Here -->
+<section class="theme-info">
+  <!-- Hero -->
+  <div class="first-section">
+  <div>
+  <h4 class="theme-intro">"Great News!"</h4>
+  <h2 class="theme">WOMEN IN TECH <br> MENTORSHIP PROGRAMME <br> 2023.</h2>
+</div>
+
+<div class="intro">
+<h4 class="intro-text">
+Are you a woman trying to navigate the tech space? then this programme is for you. join us in Rivers State for the Women In Tech mentorship program 2023 edition.
+</h4>
+</div>
+
+<div class="venue">
+<h3 class="date-text">
+6th - 7th JULY 2023
+</h3>
+<h5 class="location">
+@Landmark Hotel, Rivers State.
+</h5>
+</div>
+</div>
+
+<div class="second-section">
+  <div class="program">
+    <h4 class="main-program">Main Program</h4>
+    <hr class="one">
+  </div>
+
+
+  <div class="program-container">
+
+    <div class="program-details">
+      <img src="Assets\program_icon_01 1.png" class="detail-img" alt="Lecture"/>
+      <h5 class="detail-name">Lecture</h5>
+      <p class="detail-text">Listen to renowned speakers from various countries about the importance of mentorship in tech.</p>
+    </div>
 
-      <!-- Navbar -->
-      <div class="navbar">
-        <!-- Logo -->
+    <div class="program-details">
+      <img src="Assets\program_icon_02 1.png" class="detail-img" alt="review"/>
+      <h5 class="detail-name">Review</h5>
+      <p class="detail-text"> Sit with Recruiters from various tech companies and have your Resume reviewed.</p> 
+    </div>
 
-        <!-- Nav Items -->
+    <div class="program-details">
+      <img src="Assets\program_icon_03 1.png" class="detail-img" alt="forum"/>
+      <h5 class="detail-name">Forum</h5>
+      <p class="detail-text">Have the time to share your thoughts and opinions with experts for each topic.</p> 
+    </div>
+    <div class="program-details">
+      <img src="Assets\program_icon_04 1.png" class="detail-img" alt="workshop"/>
+      <h5 class="detail-name">Workshop</h5>
+      <p class="detail-text">Try creating your own work using open source license rather than just watching at it.</p> 
+    </div>
+    <div class="program-details">
+      <img src="Assets\program_icon_05 1.png" class="detail-img" alt="network"/>
+      <h5 class="detail-name">Network</h5>
+      <p class="detail-text">get opportunities to network with other women in tech around the world, also after the summit.</p> 
+    </div>
+    <a href="#" class="button-a">
+      <button class="button"> Click Here to Register  </button></a>
+  </div>
```

Comment:
As the previous reviewer suggested, please add the text `See the whole program` to your main program section in order to respect the behance design🙏

Your design:
<img width="948" alt="image" src="https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/37116322/acd2f1c8-e536-4d6f-8245-58414420dcfc">

Behance design:
<img width="622" alt="image" src="https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/37116322/bb5ebccd-145f-43ad-9e56-07f5e47f8854">

Reply:
Thank you so much for your review, I have added the text "SEE THE WHOLE PROGRAM" to balance the design.